.github/workflows/create-new-image.yml

```diff
@@ -0,0 +1,19 @@
+name: create-new-image
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+
+jobs:
+  create-new-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Create new docker image and push branch
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          TESTING_BUCKET=${{ secrets.TESTING_BUCKET }}
```

Comment:
I think GitHub Action's way to use the secret and ensure it is scrubbed is like this:
```
      - name: Create new docker image and push branch
        env:
          DOWNLOAD_URL: ${{ secrets.DOWNLOAD_URL }}
        run: |
          git config user.name github-actions
          git config user.email github-actions@github.com
          ./tools/update-docker-image.bash ${{ inputs.version }}
```
See https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow

Reply:
Fixed in [28c3562](https://github.com/newrelic/newrelic-php-daemon-docker/pull/75/commits/28c35622649b1a4c68a8e6ce406424224b9092d7)